tools/src/versioning/android/versionCxx/patches/expo-av.patch

```diff
@@ -0,0 +1,71 @@
+diff --git a/packages/expo-av/android/CMakeLists.txt b/packages/expo-av/android/CMakeLists.txt
+index fe625b6347..af6218de4c 100644
+--- a/packages/expo-av/android/CMakeLists.txt
++++ b/packages/expo-av/android/CMakeLists.txt
+@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.4.1)
+ set (CMAKE_VERBOSE_MAKEFILE ON)
+ set (CMAKE_CXX_STANDARD 14)
+ 
+-set (PACKAGE_NAME "expo-av")
++set (PACKAGE_NAME "expo-av_{VERSIONED_ABI_NAME}")
+ set (BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
+ 
+ # shared
+@@ -54,14 +54,14 @@ find_library(
+ 
+ find_library(
+         JSI_LIB
+-        jsi
++        jsi_{VERSIONED_ABI_NAME}
+         PATHS ${LIBRN_DIR}
+         NO_CMAKE_FIND_ROOT_PATH
+ )
+ 
+ find_library(
+         REACT_NATIVE_JNI_LIB
+-        reactnativejni
++        reactnativejni_{VERSIONED_ABI_NAME}
+         PATHS ${LIBRN_DIR}
+         NO_CMAKE_FIND_ROOT_PATH
+ )
+diff --git a/packages/expo-av/android/build.gradle b/packages/expo-av/android/build.gradle
```

Comment:
currently, when I run those scripts it will also include version libjsi and libreactnativejni in versioned jniLibs directory.

If this is not expected you can handle that in script or by adding `_{VERSIONED_ABI_NAME}` to `exlude` definitions in build.gradle patch

Reply:
@wkozyra95 that's interesting! i cannot repro the problem neither from reanimated, expo-av, nor expo-gl-cpp. anyway, i changed [the script to copy `libexpo*.so` only](https://github.com/expo/expo/pull/16959/commits/cf20abe4a19643f45db84505ae7079d74ed91cc0). this aligns to what we did in versioning `System.loadLibrary()`